src/components/SpruceForm/LeafyGreenWidgets.tsx

```diff
@@ -237,13 +236,13 @@ export const LeafyGreenRadioBox: React.FC<WidgetProps> = ({
     return null;
   }
 
-  if (rawErrors && !Array.isArray(rawErrors)) {
+  if (errors && !Array.isArray(errors)) {
     reportError(
-      new Error("LeafyGreen Radio Box expects rawErrors to be an array")
+      new Error("LeafyGreen Radio Box expects errors to be an array")
     ).warning();
     return null;
   }
-  const errs = rawErrors as string[];
+  const errs = errors as string[];
```

Comment:
Yeah exactly. I renamed from `rawErrors` to `errors` because RJSF outputs a field called `rawErrors`, so I wanted it to be clear these are errors coming from the UI schema.

I added widget option typing in #1206 though so this will get cleaned up once that's merged.